### DISCUSSION TITLE
esp32/MicroPython/04.esp32-micropython-LED

### DISCUSSION THREAD

**Discussion by giscus** [bigrich-luo/DocsGiscus#30] [General]
# esp32/MicroPython/04.esp32-micropython-LED

https://docs.geeksman.com/esp32/MicroPython/04.esp32-micropython-LED.html

**Comment by baocong2019** [comment]
楼主创作不易，为你点赞加油

**Comment by ONEtianci** [comment]
楼主创作不易，为你点赞加油
